logger/logger.go

```diff
@@ -1,30 +1,25 @@
 package logger
 
 import (
-	"bytes"
 	"encoding/json"
-	"net/http"
-	"os"
-	"time"
 
 	"github.com/joho/godotenv"
 )
 
 // Logger is the means by which a user can begin to send logs.
 type Logger struct {
-	Buffer   []Message
-	Key      string
-	Messages chan Message
-	Options  Options
+	Options Options
```

Comment:
have u considered to move all the types defined here in this file into the different file called `logger/types.go`? If that doesn't sound right, why? Just wondering...

Reply:
generally the convention is to define types in the same file where methods are defined on them. a `types.go` file isn't a pattern I've seen in other projects, and this is just a best-practice thing.